app/templates/containersgrid.html

```diff
@@ -56,6 +63,9 @@
                 <a href="/containers/{{container.tool_path}}">{{container.name + (container.toolname ? '/' + container.toolname : '')}}</a>
               </td>
               <td>
+                {{ container.starredUsers.length == 0 ? '' : container.starredUsers.length }}
```

Comment:
It should probably be either "container.starredUsers.length ? " or "container.starredUsers.length === 0 ? "